File: rumahsakitapp/lib/model/pasien.dart
class Pasien {
  final String namaPengguna;
  final String noKTP;
  final String jekel;
  final String tempatLahir;
  final String tanggalLahir;
  final String noBPJS;
  final String noHp;
  final String email;
  final String alamat;

  Pasien({
    required this.namaPengguna,
    required this.noKTP,
    required this.jekel,
    required this.tempatLahir,
    required this.tanggalLahir,
    required this.noBPJS,
    required this.noHp,
    required this.email,
    required this.alamat,
  });

  Map<String, dynamic> toJson() {
    return {
      'namaPengguna': namaPengguna,
      'noKTP': noKTP,
      'jekel': jekel,
      'tempatLahir': tempatLahir,
      'tanggalLahir': tanggalLahir,
      'noBPJS': noBPJS,
      'noHp': noHp,
      'email': email,
      'alamat': alamat,
    };
  }
}

File: RawatJalanApp/app/Models/pasien.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class pasien extends Model
{
    use HasFactory;

    protected $table = 'pasiens';

    protected $primaryKey = 'noRekapMedis';

    protected $fillable = [
        'noRekapMedis',
        'namaPengguna',
        'noKTP',
        'jekel',
        'tempatLahir',
        'tanggalLahir',
        'noBPJS',
        'noHp',
        'email',
        'alamat',
    ];

    public function noRekapMedis(): BelongsTo
    {
        return $this->belongsTo(pasien::class, 'noRekapMedis');
    }
}
